src/components/Global/TotalValue.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react'
+import { CountUp } from 'countup.js'
+
+import "./Global.css"
+
+const TotalValue = ({ value, text }) => {
+  useEffect(() => {
+    const countup = new CountUp(`countup-${text}`, value)
+    countup.start()
+  }, [])
+
+  return (
+    <h1 className="global__value">
```

Comment:
We should take care of headings hierarchy for accessibility. So we must only have one `h1` tag that is the page title. Here is an [article](https://www.w3.org/WAI/tutorials/page-structure/headings/) about headings recommended by mdn documentation.